src/sentry/snuba/metrics/extraction.py

```diff
@@ -884,7 +886,10 @@ def _remove_blacklisted_search_filters(tokens: Sequence[QueryToken]) -> Sequence
     ret_val: List[QueryToken] = []
     for token in tokens:
         if isinstance(token, SearchFilter):
-            if token.key.name not in _IGNORED_METRIC_FIELDS:
+            if (
+                token.key.name not in _IGNORED_METRIC_FIELDS
+                and str(token) not in _IGNORED_METRIC_CONDITION
+            ):
```

Comment:
@iambriccardo @obostjancic I need your eyes on this change since you originally added support for ignoring `event.type` completely.

Reply:
Looks good to me, it's a good refactor!